engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/PrivateConstructorAnalysis.java

```diff
@@ -21,31 +21,19 @@
 public final class PrivateConstructorAnalysis implements IRPass {
   public static final PrivateConstructorAnalysis INSTANCE = new PrivateConstructorAnalysis();
 
-  private UUID uuid;
-
   private PrivateConstructorAnalysis() {}
 
   @Override
-  public void org$enso$compiler$pass$IRPass$_setter_$key_$eq(UUID v) {
```

Comment:
Getting rid of these criptical `@Overrides` is very nice.

Reply:
It was a setter for some _"UUID obsession"_ not used anywhere.